ui/src/app/core/store/layout/layout.reducer.ts

```diff
@@ -1,19 +1,34 @@
 import { LayoutActions, LayoutActionTypes } from './layout.actions';
 
+export interface WindowDimensions {
+    height: number;
+    width: number;
+}
+
 export type LayoutState = Readonly<{
+    windowDimensions: WindowDimensions;
     navbarCollapsed: boolean;
     navigationCollapsed: boolean;
-    sidebarCollapsed: boolean;
+    sidebarOpen: boolean;
```

Comment:
Why not keep the naming system of *Collapsed?

Reply:
Refactoring back. Was thinking of it as more of a menu, but it is the entire sidebar.